docs/index.rst

```diff
@@ -35,18 +46,43 @@ Then run the following command to start Kafka connect with the HDFS connector::
   $ ./bin/connect-standalone etc/schema-registry/connect-avro-standalone.properties \
   etc/kafka-connect-hdfs/quickstart-hdfs.properties
 
-You should see the process start up and log some messages, and then it will export data from
-Kafka to HDFS. In order to check that the data have been copied to HDFS, you can use Hive to query
-data or Avro tools to extract the file in HDFS::
+You should see that the process starts up and logs some messages, and then it will export data from
+Kafka to HDFS. Once the connector finishes ingesting data to HDFS, check that the data is available
+in HDFS::
 
   $ hadoop fs -ls /topics/test_hdfs/partitions=0
 
 You should see a file with name ``/topics/t1/partition=0/test_hdfs+0+0000000000+0000000002.avro``
-The file name is encoded as ``topic+partition+start_offset+end_offset.format``.
+The file name is encoded as ``topic+kafkaPartition+startOffset+endOoffset.format``.
+
+You can use ``avro-tools-1.7.7.jar``
+(`<http://mirror.metrocast.net/apache/avro/avro-1.7.7/java/avro-tools-1.7.7.jar>`_)
+to extract the content of the file::
+
+  $ hadoop jar avro-tools-1.7.7.jar tojson \
+  /topics/test_hdfs/partition=0/test_hdfs+0+0000000000+0000000002.avro
+
+You should see the following output::
+
+  {"f1":"value1"}
+  {"f1":"value2"}
+  {"f1":"value3"}
+
+
+.. note:: If you want to run the Quickstart with Hive integration, before starting the connector,
+   you need to add the following configurations to
+   ``etc/kafka-connect-hdfs/quickstart-hdfs.properties``::
+
+      hive.integration=true
+      hive.metastore.uris=uri to your hive metastore
+      schema.compatibility=BACKWARD
+   After the connector finishes ingesting data to HDFS, you can use Hive to check the data::
 
-You can use ``avro-tools-1.7.7.jar`` to extract the content of the file::
+      $hive>SELECT * FROM test_hdfs;
 
-  $ hadoop jar avro-tools-1.7.7.jar tojson /topics/test_hdfs/partition=0/test_hdfs+0+0000000000+0000000002.avro
+   Note that if you leave ``hive.metastore.uris`` empty, an embedded Hive metastore will be created
```

Comment:
Yes. If the hive metastore uri is missing, the connector uses derby as the embedded metastore.